.github/workflows/codeql.yml

```diff
@@ -0,0 +1,60 @@
+
+name: "CodeQL"
+
+on:
+  push:
+    branches: [ develop ]
+  pull_request:
+    # The branches below must be a subset of the branches above
+    branches: [ develop ]
+  schedule:
+    - cron: '37 8 * * 6'
```

Comment:
Let’s disable cron jobs because pushes and pull requests should do the job.

Reply:
I have removed the cron Job.